d3a_api_client/redis_device.py

```diff
@@ -32,10 +39,41 @@ def _subscribe_to_response_channels(self):
         channel_subs[f'{self._channel_prefix}/events/tick'] = self._on_tick
         channel_subs[f'{self._channel_prefix}/events/trade'] = self._on_trade
         channel_subs[f'{self._channel_prefix}/events/finish'] = self._on_finish
+        channel_subs["crud_aggregator_response"] = self._aggregator_response_callback
```

Comment:
Can you tell me the reason for calling it `crud_aggregator`? I looked it up and it means "dirt" is this really what you meant?

Reply:
CRUD - Create, Read, Update & Delete is a typical short-form in CS. Since the purpose of this channel is add/delete aggregator as well as selection of aggregator locally, so I added it as a prefix, though Im open to better naming suggestion

Reply:
Why not simply "aggregator"?